README.md

```diff
@@ -105,39 +121,28 @@ Create the following services:
 * [**Watson Speech To Text**](https://console.bluemix.net/catalog/services/speech-to-text)
 * [**Watson Text To Speech**](https://console.bluemix.net/catalog/services/text-to-speech)
 
-### 3. Import the Conversation workspace
+### 3. Upload the Watson Assistant workspace
 
-Launch the `Watson Assistant` tool. Use the `import` icon button on the right
+Now that our services are created and the app is deployed we need to update tell the application to use a specific Watson Assistant dialog. We'll be using the file [`data/workspace.json`](data/workspace.json), which documents our entire conversation dialog. To do this, launch the Watson Assistant tool and use the `import` icon button on the right. Find the [`data/workspace.json`](data/workspace.json) file from the cloned repo and import that to the Watson Assistant tool.
```

Comment:
the app is deployed we need to update the application ...

Reply:
done